src/pages/VeSDL/VeTokenCalculator.tsx

```diff
@@ -99,14 +129,31 @@ export default function VeTokenCalculator({
 
           <TextField
             label={t("poolLiquidity")}
-            value={formatBNToString(totalLPAmount, 18)}
+            value={userLPAmountInput}
+            onChange={(e) => {
+              if (
+                readableDecimalNumberRegex.test(e.target.value) ||
+                e.target.value === ""
+              ) {
+                setUserLPAmountInput(e.target.value)
+              }
+            }}
           />
           <TextField
             label={t("depositAmount")}
-            value={formatBNToString(userLPAmount, 18)}
+            value={totalLPAmountInput}
+            onChange={(e) => {
+              if (
+                readableDecimalNumberRegex.test(e.target.value) ||
+                e.target.value === ""
+              ) {
+                setTotalLPAmountInput(e.target.value)
+              }
+            }}
           />
           <Typography>
-            {t("maxBoost")}: {MAX_BOOST}
+            {t("maxBoost")}:{" "}
```

Comment:
Instead of a " " space, can you put the values into seperate Typography components and add padding/margin?